src/services/expenses.service.js

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable function-paren-newline */
+const { Expense } = require('../models/Expense.model');
+
+const allExpenses = async (userId, categories, from, to) => {
+  try {
+    const filter = {};
+
+    if (userId) {
+      filter.userId = userId;
+    }
+
+    if (categories) {
+      filter.category = categories;
+    }
+
+    if (from || to) {
+      filter.spentAt = {};
+
+      if (from) {
+        filter.spentAt['$gte'] = from;
+      }
+
+      if (to) {
+        filter.spentAt['$lte'] = to;
+      }
+    }
+
+    const expenses = await Expense.findAll({ where: filter });
+
+    return expenses;
+  } catch (error) {
+    return error;
+  }
```

Comment:
You don't need to wrap with try/catch and it's a bad idea to return errors, you always need to throw them